src/stores/nav.js

```diff
@@ -0,0 +1,28 @@
+import { defineStore } from 'pinia'
+import { reactive, toRefs } from '@nuxtjs/composition-api'
+
+export const NAV = 'nav'
```

Comment:
I think we decided we didn't need to export these.

```suggestion
const NAV = 'nav'
```